app/controllers/admin/participants_controller.rb

```diff
@@ -2,26 +2,34 @@
 
 module Admin
   class ParticipantsController < Admin::BaseController
-    skip_after_action :verify_policy_scoped, only: :show
+    skip_after_action :verify_policy_scoped, except: :index
     skip_after_action :verify_authorized, only: :index
 
-    before_action :load_participant, only: :show
+    before_action :load_participant, except: :index
 
     def show; end
 
     def index
       school_cohort_ids = SchoolCohort.ransack(school_name_or_school_urn_cont: params[:query]).result.pluck(:id)
       query = "%#{(params[:query] || '').downcase}%"
       @participant_profiles = policy_scope(ParticipantProfile).joins(:user)
-                                  .where("lower(users.full_name) LIKE ? OR school_cohort_id IN (?)", query, school_cohort_ids)
-                                  .order("DATE(users.created_at) asc, users.full_name")
+                                                              .active
+                                                              .where("lower(users.full_name) LIKE ? OR school_cohort_id IN (?)", query, school_cohort_ids)
+                                                              .order("DATE(users.created_at) asc, users.full_name")
+    end
+
+    def remove; end
+
+    def destroy
+      @participant_profile.withdrawn!
+      render :destroy_success
     end
 
   private
 
     def load_participant
-      @participant = User.is_participant.find(params[:id])
-      authorize @participant, policy_class: ParticipantPolicy
+      @participant_profile = ParticipantProfile.find(params[:id])
+      authorize @participant_profile, policy_class: ParticipantProfilePolicy
```

Comment:
Do we need to specify the class here?

Reply:
FYI @BroiSatse I think these are on the cherry picked commit. Will change these, but you might get a few conflicts

Reply:
Yes, this is needed
`Pundit::NotDefinedError: unable to find policy `ParticipantProfile::MentorPolicy`